File: java-helidon-nima-ms/nima/src/main/java/com/maocq/helidon/nima/application/Application.java
package com.maocq.helidon.nima.application;

import com.maocq.helidon.nima.application.config.ConfigApp;
import com.maocq.helidon.nima.domain.usecases.cases.CasesUseCase;
import com.maocq.helidon.nima.infrastructure.drivenadapters.httpclient.HelloHttpClient;
import com.maocq.helidon.nima.infrastructure.entrypoints.RestController;

public class Application {

    public static void main(String[] args) {;
        var httpClient = ConfigApp.getHttpClient();
        var cases = new CasesUseCase(new HelloHttpClient(httpClient));

        new RestController(cases).start();
    }
}

File: go-ms/application/application.go
package application

import (
	"maocq/go-ms/application/config"
	"maocq/go-ms/domain/usecase"
	"maocq/go-ms/infrastructure/driven-adapters/httpclient"
	"maocq/go-ms/infrastructure/entry-points/rest"
)

func Start() {
	httpClient := config.GetHttpClient()

	helloRepository := httpclient.HelloHttpRepository{Client: httpClient, Url: config.GetUrlService()}

	cases := usecase.CasesUseCase{HelloRepository: &helloRepository}

	rest.Start(&cases)
}
